RadiationForce_using_FIR_Filter/wecSimInputFile.m

```diff
@@ -0,0 +1,78 @@
+%% Simulation Data
+simu = simulationClass();               % Initialize Simulation Class
+simu.simMechanicsFile = 'RM3.slx';      % Specify Simulink Model File
+simu.mode = 'normal';                   % Specify Simulation Mode ('normal','accelerator','rapid-accelerator')
+simu.explorer = 'off';                   % Turn SimMechanics Explorer (on/off)
+simu.startTime = 0;                     % Simulation Start Time [s]
+simu.rampTime = 100;                    % Wave Ramp Time [s]
+simu.endTime = 1000;                     % Simulation End Time [s]
```

Comment:
@salhus Minor comment: Suggest reducing to 500 s as the 1000 s makes the automatic plot look very condensed and need to maximize the plot to see the wave cycles and easier see differences between radiation options.

Reply:
- done